packages/backend/src/handlers/auth-webhook/users.ts

```diff
@@ -4,24 +4,64 @@ async function createPlayer(ethAddress: string) {
   const { insert_profile: insert } = await client.CreatePlayerFromETH({
     ethereumAddress: ethAddress,
   });
-  if (insert?.affected_rows !== 1) {
-    throw new Error('Error while creating player');
+  if (insert?.affected_rows !== 2) {
```

Comment:
how come the condition for throwing an error changed from `insert?.affected_rows !== 1` to `insert?.affected_rows !== 2`?